commodore/cluster.py

```diff
@@ -59,7 +61,7 @@ def _full_target(cluster, components, catalog):
                 'catalog_url': f"{catalog}",
             },
             'cloud': {
-                'type': f"{cloud_provider}",
+                'provider': f"{cloud_provider}",
```

Comment:
Have you verified that all components expect and use `cloud.provider` rather than `cloud.type`?

Reply:
Yes, and I changed the one occurrence I found.